.github/workflows/ci.yaml

```diff
@@ -163,60 +163,78 @@ jobs:
         working-directory: ./fe2-android
 
     steps:
-    - name: Setup repo
-      uses: actions/checkout@v2
-      with:
-        submodules: recursive
-        fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of Sonar analysis
-
-    - name: Setup JDK 11
-      uses: actions/setup-java@v1
-      with:
-        java-version: 11.0.10
-
-    - name: Cache SonarCloud packages
-      uses: actions/cache@v1
-      with:
-        path: ~/.sonar/cache
-        key: ${{ runner.os }}-sonar
-        restore-keys: ${{ runner.os }}-sonar
-
-    - name: Cache Gradle packages
-      uses: actions/cache@v1
-      with:
-        path: ~/.gradle/caches
-        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
-        restore-keys: ${{ runner.os }}-gradle
-
-    - name: Grant execute permission for gradlew
-      run: |
-        chmod +x ./gradlew
+      - name: Checkout
+        uses: actions/checkout@v2
+        
+      - name: Setup JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'  
 
-    - name: Build
-      shell: bash
-      run: |
-        # use grep to remove log spam due to jetifier transformation, c.f. https://stackoverflow.com/a/52353446
-        # use set -o pipefail to keep error code from gradle, c.f. https://stackoverflow.com/a/37128767
-        ( set -o pipefail; ./gradlew build --parallel --build-cache --info | grep -vE "^Transforming (.*) with (.*)Transform" )
+      - name: Gradle cache
```

Comment:
I'd still try to keep verbs rather than nouns here (and below), e.g. `Cache X packages` or `Cache AVD` rather than `X cache`